bundles/org.openhab.binding.plex/src/main/java/org/openhab/binding/plex/internal/handler/PlexServerHandler.java

```diff
@@ -131,11 +131,18 @@ public void initialize() {
             }
         }
         logger.debug("Fetch API with config, {}", config.toString());
-        if (!plexAPIConnector.getApi()) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                    "Unable to fetch API, token may be wrong?");
+        try {
+            if (!plexAPIConnector.getApi()) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "Unable to find server, make sure `server` is present in Plex XML response.");
```

Comment:
Maybe a bit technical. I think it is better to rephrase. _"...make sure `server` matches the Plex network configuration."_